amoro-ams/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -380,6 +380,12 @@ public class AmoroManagementConf {
           .defaultValue(Duration.ofSeconds(3))
           .withDescription("Optimizer polling task timeout.");
 
+  public static final ConfigOption<Duration> OPTIMIZER_QUEUE_REFRESH_INTERVAL =
+      ConfigOptions.key("optimizer.queue-refresh-interval")
```

Comment:
```suggestion
      ConfigOptions.key("optimizing-service.refresh-group-interval")
```

Reply:
WDYT? @zhoujinsong

Reply:
The name `self-optimizing.refresh-group-interval` seems appropriate, especially since we already have some configurations prefixed with `self-optimizing`.

Reply:
Besides, can you help to add this new configuration example to the configuration temp file.